tests/testutils/telemetry/logs.go

```diff
@@ -341,3 +348,38 @@ func (resourceLogs ResourceLogs) ContainsAll(expected ResourceLogs) (bool, error
 	}
 	return true, nil
 }
+
+func (resourceLogs ResourceLogs) StartsWith(expected ResourceLogs) (bool, error) {
```

Comment:
Given the sparsity of the exercised resource logs I don't think this assertion helper is desirable. There are already systems to relax the equivalence checks like not including fields in the log definition and `<ANY>` for attributes. I request adding new symbols or regex support for your specific cases and increasing the specificity of the expected logs.

Reply:
I'm happy to help with this in a separate change set, and if you were to provide examples of the complete resource logs for a single script I'd be more up to speed.

Reply:
Please find below example output of df.sh execution, is it what you asked for?
Filesystem	Size	Used	Avail	Capacity	iused	ifree	IUsePct	MountedOn	Type	INodes
disk1s1	466Gi	12Gi	47Gi	21%	349475	496486880	0%	/	apfs	496836355
disk1	466Gi	3.2Gi	47Gi	7%	1265	496486880	0%	/System/Volumes/Preboot	apfs	496488145
disk1	466Gi	6.0Gi	47Gi	12%	6	496486880	0%	/System/Volumes/VM	apfs	496486886
disk1	466Gi	58Mi	47Gi	1%	721	496486880	0%	/System/Volumes/Update	apfs	496487601
disk1	466Gi	396Gi	47Gi	90%	3690593	496486880	1%	/System/Volumes/Data	apfs	500177473
disk1	466Gi	12Gi	47Gi	21%	355386	496486880	0%	/System/Volumes/Update/mnt1	apfs	496842266

Reply:
This message is piped as one entry and Splunk with Addon for Unix and Linux will split it appropriately and extract required fields.

Reply:
The regexp that would check beginning of the message for appropriate header would suffice I think.